src/constant/index.ts

```diff
@@ -1,38 +1,55 @@
 import IconHecoLogo from 'ui/assets/chain-logos/heco.svg';
+import IconHecoChainLogo from 'ui/assets/dashboard/selectChain/heco.svg';
 import IconHecoWhiteLogo from 'ui/assets/chain-logos/heco-white.svg';
 import IconBscLogo from 'ui/assets/chain-logos/bsc.svg';
 import IconBscWhiteLogo from 'ui/assets/chain-logos/bsc-white.svg';
+import IconBscChainLogo from 'ui/assets/dashboard/selectChain/bsc.svg';
 import IconDaiLogo from 'ui/assets/chain-logos/dai.svg';
 import IconDaiWhiteLogo from 'ui/assets/chain-logos/dai-white.svg';
+import IconDaiChainLogo from 'ui/assets/dashboard/selectChain/xdai.svg';
 import IconPolygonLogo from 'ui/assets/chain-logos/polygon.svg';
 import IconPolygonWhiteLogo from 'ui/assets/chain-logos/polygon-white.svg';
+import IconPolygonChainLogo from 'ui/assets/dashboard/selectChain/polygon.svg';
 import IconFantom from 'ui/assets/chain-logos/fantom.svg';
 import IconFantomWhiteLogo from 'ui/assets/chain-logos/fantom-white.svg';
+import IconFantomChainLogo from 'ui/assets/dashboard/selectChain/fantom.svg';
 import IconOKTLogo from 'ui/assets/chain-logos/okex.svg';
 import IconOKTWhiteLogo from 'ui/assets/chain-logos/okex-white.svg';
+import IconOKTChainLogo from 'ui/assets/dashboard/selectChain/okex.svg';
 import IconMetisLogo from 'ui/assets/chain-logos/metis.svg';
 import IconMetisWhiteLogo from 'ui/assets/chain-logos/metis-white.svg';
+import IconMetisChainLogo from 'ui/assets/dashboard/selectChain/metis.svg';
 import IconArbitrumLogo from 'ui/assets/chain-logos/arbitrum.svg';
 import IconArbitrumWhiteLogo from 'ui/assets/chain-logos/arbitrum-white.svg';
+import IconArbitrumChainLogo from 'ui/assets/dashboard/selectChain/arbitrum.svg';
 import IconOPLogo from 'ui/assets/chain-logos/op.svg';
 import IconOPWhiteLogo from 'ui/assets/chain-logos/op-white.svg';
+import IconOPChainLogo from 'ui/assets/dashboard/selectChain/optimistic.svg';
 import IconAvaxLogo from 'ui/assets/chain-logos/avax.svg';
 import IconAvaxWhiteLogo from 'ui/assets/chain-logos/avax-white.svg';
+import IconAvaxChainLogo from 'ui/assets/dashboard/selectChain/avalanche.svg';
 import IconCeloLogo from 'ui/assets/chain-logos/celo.svg';
 import IconCeloWhiteLogo from 'ui/assets/chain-logos/celo-white.svg';
+import IconCeloChainLogo from 'ui/assets/dashboard/selectChain/celo.svg';
 import IconMoonriverLogo from 'ui/assets/chain-logos/movr.svg';
 import IconMoonriverWhiteLogo from 'ui/assets/chain-logos/movr-white.svg';
+import IconMoonriverChainLogo from 'ui/assets/dashboard/selectChain/moonriver.svg';
 import IconCronosLogo from 'ui/assets/chain-logos/cronos.svg';
 import IconCronosWhiteLogo from 'ui/assets/chain-logos/cronos-white.svg';
+import IconCronosChainLogo from 'ui/assets/dashboard/selectChain/cronos.svg';
 import IconBobaLogo from 'ui/assets/chain-logos/boba.png';
 import IconBobaWhiteLogo from 'ui/assets/chain-logos/boba-white.svg';
+import IconBobaChainLogo from 'ui/assets/dashboard/selectChain/bola.svg';
```

Comment:
can you rename `bola.svg` to `boba.svg`?

Reply:
done